apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -85,6 +85,21 @@ can in inject the `IManagedTracer` into the action method using the `[FromServic
 
 {{sample:Trace.UseTracerRunIn}}
 
-## Trace Outgoing HTTP Requests
+## Trace Outgoing HTTP Requests (all platforms)
 
 {{sample:Trace.TraceOutgoing}}
+
+## Trace Outgoing HTTP Requests (recommend for Asp.Net Core 2.0 upwards)
+
+The [recommend way of creating HTTPClient](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1) in Asp.Net Core 2.0 and upwards is to use the 
+`System.Net.Http.IHttpClientFactory` defined in the Microsoft.Extensions.Http package.
+See the following example on how to register and use an HttpClient using Google Trace so that it traces
```

Comment:
"The following example demonstrates how to"?

Reply:
Done